client-web/src/exlibris-app/exlibris-app.html

```diff
@@ -174,14 +256,20 @@
              * @private
              */
             _getTitle(page) {
-                let title;
-                if (page) {
-                    const unDashedString = page.replace(/-/g, ' ');
-                    title = unDashedString.replace(/\w\S*/g, function (txt) {
-                        return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
-                    });
+                switch (page) {
+                    case "all-books":
+                        return "All Books";
+                    case"expected-soon":
+                        return "Expected Soon";
+                    case "borrowed-books":
+                        return "Borrowed Books";
+                    case "reserved-books":
+                        return "Reserved Books";
+                    case "should-return-soon":
+                        return "Should Return Soon";
+                    case"overdue-books":
+                        return "Overdue Books";
```

Comment:
It can be replaced with property like:
```
titles: {
   ['all-books']: 'All Books',
...
}
```
and use it in template.

Reply:
@dkalinin  I created the next property : 
```
                 titles: {
                        type: Object,
                        value: {
                            ['all-books']: 'All Books',
                            ['expected-soon']: 'Expected Soon',
                            ['borrowed-books']: 'Borrowed Books',
                            ['reserved-books']: 'Reserved Books',
                            ['should-return-soon']: 'Should Return Soon',
                            ['overdue-books']: 'Overdue Books'
                        }
                    }
```
How do I correctly display a title ? `[[titles[[[routeData.page]]]]]` shows `[[titles[all-books]]]`

Reply:
You need to call method anyway.